### DISCUSSION TITLE
私有registry中发布theme的包如何命名

### DISCUSSION THREAD

**Discussion by lemuellew** [slidevjs/slidev#637] [Q&A] [answered]
slidev自动识别theme的是需要包名为@slidev/theme-xxx或者slidev-theme-xxx，如果我想在私有registry（发布包名要求必须带@scope）中发布一个theme，请问有什么好的方式吗？

**Comment by yjl9903** [comment, marked as answer]
见 #604 #605 , 应该可以直接写 @xxx/theme-yyy
